### ISSUE TITLE
How to change from email/password to pin number in login?

### ISSUE THREAD

**Issue by isaacdarcilla** [bailabs/tailpos#88]

**Comment by Dobbs3313** [comment]
Yeah, I am having the same issue. Some devices seems to go straight to the pin number login while others launch with the email/password form first...

**Comment by isaacdarcilla** [comment]
Why is that?

**Comment by sammish** [comment]
@isaacdarcilla @Dobbs3313 please mail me I will support you this case. sammish.thundiyil@gmail.com
